CHANGELOG.md

```diff
@@ -27,4 +27,6 @@ Headings should be (only add headings as needed):
 - Fixed typo in paypal pro ([535](https://github.com/eventespresso/event-espresso-core/pull/535))
 - Fixed EE CPT objects not being added to WP_Post objects during Ajax Requests ([531](https://github.com/eventespresso/event-espresso-core/pull/531))
 
+### Changed
 
+- Link payment logs to the transaction if an EE_Transaction object is passed to the log call (545(https://github.com/eventespresso/event-espresso-core/pull/545))
```

Comment:
I think this should go in the fixed section, its a bug right?  Also please use declarative (or indicative) mood, you can see docs on our changelog entries [here](https://github.com/eventespresso/event-espresso-core/blob/master/docs/A--Best-Practices/change-log.md)

Finally, your formatting for the link to the pull is incorrect. You can get the correct format from the other changelog entries.

Thanks!